### PR TITLE
Update paid-time-off.md

### PR DESCRIPTION
**Pull request by lynn915**

### CODE REVIEW

**Review by natalie-hub — DISMISSED** [review]
Propose we update the wording "if there are a lot of Entity Staff in the meeting" to "if there are a lot of Staff that cannot attend."

**Comment by natalie-hub** [comment]
I made one proposed change in my comment, but I forgot to tag @lynn915 and @jasonblais.

File: operations/workplace/people/working-at-mattermost/paid-time-off.md [hunk 1]
```diff
@@ -39,31 +39,38 @@ Note: If you are taking [Pregnancy](leaves-of-absence/pregnancy-leave.md) or [Pa
 
 Please take off holidays relevant to your culture, resident [country/region](../../../../company/about-mattermost/list-of-terms.md#country-region) and preferences. When doing so, please follow the time-off process in [Communicating Time Off](paid-time-off.md#communicating-time-off).
```
Comment (justinegeffen) [inline operations/workplace/people/working-at-mattermost/paid-time-off.md]:
```suggestion
Please take off holidays relevant to your culture, resident [country/region](../../../../company/about-mattermost/list-of-terms.md#country-region) and preferences. When doing so, please follow the time off process in [Communicating Time Off](paid-time-off.md#communicating-time-off).
```

File: operations/workplace/people/working-at-mattermost/paid-time-off.md [hunk 2]
```diff
@@ -39,31 +39,38 @@ Note: If you are taking [Pregnancy](leaves-of-absence/pregnancy-leave.md) or [Pa
...
+* Independence Day (US) \(7/3/2020\)
+* Civic Holiday/Provincial Day (Canada) \(8/3/2020\)
+* Labor Day (US, Canada) \(9/7/2020\)
+* Thanksgiving (Canada) \(10/12/2020\)
+* Rememberance Day (Canada: BC, AB, PEI) \(11/11/2020\)
+* Thanksgiving (US) \(11/26/2020\)
+* Christmas Day (All) \(12/25/2020\)
+* Boxing Day (Canada: ON, Germany, UK) \(12/28/2020\)
+
+It’s up to meeting chairs to decide whether to reschedule if few people can attend \(e.g. if there are a lot of Entity Staff in the meeting\).
```
Comment (justinegeffen) [inline operations/workplace/people/working-at-mattermost/paid-time-off.md]:
```suggestion
It’s up to meeting chairs to decide whether to reschedule if few people can attend \(e.g. if a large number of staff will not be attending the meeting\).
```

**Review by justinegeffen — DISMISSED** [review]
One suggestion for rewording, but otherwise LGTM! Thanks. :)

**Comment by natalie-hub** [comment]
Thanks @justinegeffen that change looks good. I resolved it. I let @lynn915 know that I "resolved" the update so this is good to go. Thank you!

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**